mix_helpers.exs

```diff
@@ -252,13 +252,26 @@ defmodule Mix.Appsignal.Helper do
   end
 
   defp download_options do
-    options = [
-      ssl_options:
+    default_ssl_options = tls_options() ++ customize_hostname_check_or_verify_fun()
+    default_cacert_file_path = priv_path("cacert.pem")
+
+    options =
+      if File.exists?(default_cacert_file_path) do
```

Comment:
Can we also include a "readable" check? Not just that file exists, but that it can be read (by the current process).